### ISSUE TITLE
Get Jenkins tests working on Gadi

### ISSUE THREAD

**Issue by nichannah** [COSIMA/access-om2#182]
Currently all automated tests (except for PEP8) rely on a connection to Raijin. this needs to be updated for the new machine.

**Comment by aidanheerdegen** [comment]
Do you want me to have a crack at fixing this up @nichannah?

**Comment by aidanheerdegen** [comment]
FYI I am looking into this now.

**Comment by aekiss** [comment]
@aidanheerdegen can this be closed now?

**Comment by aidanheerdegen** [comment]
Yeah, although ironically some have failed again because `gadi` is down. 

The tools build is still offline, but if you're happy to not worry about that we can close this.

**Comment by aekiss** [comment]
I don't have an opinion either way on the tools build. Seems like it was last working on raijin, in Feb 2019, so the failures pre-date the move to gadi. Wondering about cost/benefit - does it look like a hard thing to get working?
